docs/ptz-camera.md

```diff
@@ -0,0 +1,129 @@
+---
+title: RTZ camera control in Home Assistant
+contributors: [nakata5321]
+---
+
+In this article, you will set up RTZ camera in Home Assistant. 
+To set up RTZ camera in Home assistant you will use ONVIF protocol. 
+To use it usually you have to set up local camera account, his process won't be covered in this article.
```

Comment:
this process

Reply:
and maybe highlight the warning that the process of setting up the local camera account is not going to be covered in the article